app/serializers/participant_declaration_serializer.rb

```diff
@@ -10,7 +10,7 @@ class ParticipantDeclarationSerializer
   set_type :'participant-declaration'
   attributes :participant_id, :declaration_type, :course_identifier, :voided
 
-  attribute :eligible_for_payment do |declaration|
+  attribute :participant_eligible_for_funding do |declaration|
```

Comment:
That does not sound like it wants to be in this PR, right?

Reply:
oops, was looking at something else

Reply:
will roll back this commit